### ISSUE TITLE
Add support for multi-arch

### ISSUE THREAD

**Issue by JenswBE** [panubo/docker-sshd#59]
Thanks for the great Docker image! Unfortunately, I noticed the image is only being built for amd64. It would be nice to also have images for e.g. arm/v7, so it can run on a Raspberry Pi as well.

Luckily the base image supports quite some architectures. Adding other arches should be quite easy:
- [GitHub Actions example](https://github.com/JenswBE/python-dead-link-checker/blob/48ee447419f571fe88b4f93807a42da363f11ff6/.github/workflows/test-build-publish.yml)
- [Native example](https://github.com/JenswBE/python-dead-link-checker/blob/75dd4122b55f0ff4239ea00d84c1be0b5c3cb1a0/.github/workflows/test-build-publish.yml)

Do you see any possibility to support this?

**Comment by macropin** [comment]
Yes, happy to support this. Please feel free to send a PR.

**Comment by JenswBE** [comment]
I just checked how I could implement this for your repo, but I noticed currently you're using a `Makefile` to publish to DockerHub. If you want to keep it as-is, I could implement the ["native" way](https://github.com/JenswBE/python-dead-link-checker/blob/75dd4122b55f0ff4239ea00d84c1be0b5c3cb1a0/.github/workflows/test-build-publish.yml). Unfortunately, this means a recent version of Docker should be installed locally with experimental features enabled to publish from local or another CI/CD.

Since this is a public repo, [GitHub Actions could be used for free](https://github.com/pricing). Is using GitHub Actions as CI/CD to build and publish to DockerHub something you might consider or definitely a no-go? Because in this case, multi-arch builds can be very easily setup using predefined actions.
